plugin.video.catchuptvandmore/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.catchuptvandmore"
        name="Catch-up TV &amp; More"
-       version="0.2.16"
+       version="0.2.17"
```

Comment:
please add the resource addon as a dependency and make the main addon use the images from the resources addon

Reply:
Yes no problem, I will do when the resource add-on will be merged 😉

Reply:
Resource addon has been aproved so it will be merged when an addon actually depends on it. But since no addon is using it we can't merge it right now :)
So please adjust on this side so we can merge on the other

Reply:
Done